lib/features/family/features/home_screen/presentation/pages/family_home_screen.dart

```diff
@@ -0,0 +1,129 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:givt_app/features/auth/cubit/auth_cubit.dart';
+import 'package:givt_app/features/family/app/family_pages.dart';
+import 'package:givt_app/features/family/features/home_screen/widgets/give_button.dart';
+import 'package:givt_app/features/family/features/home_screen/widgets/gratitude_game_button.dart';
+import 'package:givt_app/features/family/features/profiles/cubit/profiles_cubit.dart';
+import 'package:givt_app/features/family/features/profiles/widgets/profiles_empty_state_widget.dart';
+import 'package:givt_app/features/family/features/reflect/presentation/models/grateful_avatar_bar_uimodel.dart';
+import 'package:givt_app/features/family/features/reflect/presentation/models/grateful_avatar_uimodel.dart';
+import 'package:givt_app/features/family/features/reflect/presentation/widgets/grateful_avatar_bar.dart';
+import 'package:givt_app/features/family/shared/design/components/components.dart';
+import 'package:givt_app/features/family/shared/widgets/loading/custom_progress_indicator.dart';
+import 'package:givt_app/features/family/shared/widgets/texts/shared_texts.dart';
+import 'package:givt_app/shared/widgets/fun_scaffold.dart';
+import 'package:go_router/go_router.dart';
+
+class FamilyHomeScreen extends StatelessWidget {
+  const FamilyHomeScreen({super.key});
+  @override
+  Widget build(BuildContext context) {
+    final name = context.read<AuthCubit>().state.user.lastName;
+    return BlocBuilder<ProfilesCubit, ProfilesState>(
```

Comment:
Please let's not use ProfilesCubit but create a HomeScreenCubit for this screen. ProfilesCubit is a mess and eventually I want to remove it all together.